lib/auth/bot_test.go

```diff
@@ -97,3 +230,145 @@ func TestBotOnboardFeatureDisabled(t *testing.T) {
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "not licensed")
 }
+
+func renewBotCerts(
+	srv *TestTLSServer, tlsCert tls.Certificate, botUser string,
+	publicKey []byte, privateKey []byte,
+) (*Client, *proto.Certs, tls.Certificate, error) {
+	client := srv.NewClientWithCert(tlsCert)
+
+	certs, err := client.GenerateUserCerts(context.Background(), proto.UserCertsRequest{
```

Comment:
Can we pass the context as an argument? Otherwise, this call cannot be canceled.

Reply:
https://github.com/gravitational/teleport/pull/15117/commits/3064ed7c6b9558fe8d43c53900bd62582f610ca4